app/forms/steps/client/has_nino_form.rb

```diff
@@ -7,14 +7,22 @@ class HasNinoForm < Steps::BaseFormObject
       NINO_REGEXP = /\A(?!BG)(?!GB)(?!NK)(?!KN)(?!TN)(?!NT)(?!ZZ)[A-CEGHJ-PR-TW-Z][A-CEGHJ-NPR-TW-Z][0-9]{6}([A-DFM])\z/
 
       attribute :nino, :string
-      validates :nino, format: { with: NINO_REGEXP }
+      validate :validate_nino
 
       def nino=(str)
         super(str.upcase.delete(' ')) if str
       end
 
       private
 
+      def validate_nino
+        if nino.blank?
+          crime_application.not_means_tested? ? return : errors.add(:nino, :blank)
```

Comment:
NINO remains mandatory for means tested cases as we still need it to run Benefit Checker, but it can be optional if the application is not means tested